src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/highlighted.html.twig

```diff
@@ -0,0 +1,34 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
```

Comment:
Looks identical to ` src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/badge.html.twig` 😄

Reply:
Very similar indeed the only difference it does not highlight the abandoned cart with a badge (not sure if it;s important though):

![image](https://user-images.githubusercontent.com/32385991/73730632-c6ad9d80-473f-11ea-9eef-e1705a892dc9.png)

Reply:
Answered previously